### ISSUE TITLE
Calendar Fatal error in master branch commit 24e2ce0

### ISSUE THREAD

**Issue by tmccormi** [LibreHealthIO/lh-ehr#1249]
Getting a fatal error related to izmodal on my Ubuntu 16.04LTS
![calendar-notworking](https://user-images.githubusercontent.com/285835/46497539-ba25d780-c7cf-11e8-9617-6a3b9a49b8c4.png)

**Comment by tmccormi** [comment]
There is no "css" directory in the interface directory ... (and there shouldn't be)
GET http://localhost/lh-ehr-dev/interface/css/userdata.css net::ERR_ABORTED 404 (Not Found)

code:
```
interface/main/tabs/templates/user_data_template.php
13:<link rel="stylesheet" type="text/css" href="../../css/userdata.css"/>
```
Seems like the relative path is getting borked somewhere.

**Comment by tmccormi** [comment]
@aethelwulffe you were the last person to touch that particular line of code... :-)

**Comment by tmccormi** [comment]
find . -name userdata.css
./interface/main/tabs/css/userdata.css

**Comment by aethelwulffe** [comment]
So you are saying the move of templates to /sites.  Gotcha.  Checking.

**Comment by aethelwulffe** [comment]
Actually, never noticed this stuff.  Guess I should look and see why it would seem like I touched it in the past.  Don't recall.

**Comment by aethelwulffe** [comment]
So, originally when I noticed the error, the line was .../../css/userdata.css
I looked at it, and said "oops, someone gots too many dot thingies!  Delete one, good to go!
Wrong.

**Comment by aethelwulffe** [comment]
that was #1150

**Comment by aethelwulffe** [comment]
Third dot had been introduced in https://github.com/LibreHealthIO/lh-ehr/commit/8f381b1cea811d07abf830849bc84067d8222883#diff-37bf90ba2e40d4f1442aa9d79a04a24a

#653 sort of indicates that this was a boo-boo in the tabs code from day one....???

**Comment by tmccormi** [comment]
Path is fixed the real problem (with calendar) appears to be in the module drag and drop code by @apooravc 

![index-token](https://user-images.githubusercontent.com/285835/46503302-54414c00-c7df-11e8-97f9-42153b8f0db5.png)

**Comment by tmccormi** [comment]
I don't see anything obvious in this bit of code
![drag_copy](https://user-images.githubusercontent.com/285835/46503628-34f6ee80-c7e0-11e8-9171-505571109a72.png)

**Comment by tmccormi** [comment]
I think since it was css, that it failed (all along) and I only found it because I was looking that to Console debugger for the real problem.

**Comment by aethelwulffe** [comment]
I looked at all the .ajax and all the stuff in the jsonData variable data packet.  I don't see any issues there so far.

**Comment by aethelwulffe** [comment]
error status function defined more than once.  I don't understand the scope (private etc...) so not sure if that could result in a parse error.

**Comment by aethelwulffe** [comment]
There is a default value for errorString if one of the preceding conditional statements does not fire.  I don't know if there is a type restriction on the value for status and error.  In other languages, I would expect (pseudocode): 
`if (isstring(status)) {errorString .=string(status);}` but know nothing about no JS really.

**Comment by aethelwulffe** [comment]
Non-ascii characters in the code, at least in comments:  eventâ€™s

**Comment by tmccormi** [comment]
I'll try and get one of them js geeks I have to take a look.

**Comment by tmccormi** [comment]
Is anyone else having the problem I posted about, ie: the calendar is blank (first screen shot).. presumably because of this syntax error?  @teryhill @aethelwulffe

**Comment by tmccormi** [comment]
I get the same result in Chrome or FireFox.  This is the firefox cosole message

```
WARNING: An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing main_info.php:46

SyntaxError: expected expression, got ')'[Learn More] index.php:483:5 
```

**Comment by tmccormi** [comment]
As a test, I reverted the to the previous commit and the problem goes away 
... so this commit is broken @apooravc 
24e2ce0 Event drag-resize-copy feature (#1199)

**Comment by aethelwulffe** [comment]
Revert.  Must have been the last commit somehow.  It sure is cool when 
it is working!  I never had *one* issue.

**Comment by tmccormi** [comment]
Done:  
git reset HEAD^ --hard
HEAD is now at 808e46f change modal refresh (#1206)
tony@DragonFire:/opt/gitrepos/lh-ehr (master)$ git push origin -f
Total 0 (delta 0), reused 0 (delta 0)
To https://github.com/LibreHealthIO/lh-ehr.git
 + 5505095...808e46f master -> master (forced update)

**Comment by tmccormi** [comment]
I'll reapply the minor CSS fix in a separate commit

**Comment by teryhill** [comment]
The Drag and Drop version I have is working great on my system and a client system. I am not sure what the issues are and it will be some time before I can test the repo code. Reverting it is most likely the best thing until.........

**Comment by tmccormi** [comment]
This is back again.  I had removed/reverted this commit and now it in the master again...  not sure why.
24e2ce0 Event drag-resize-copy feature (#1199)
M       library/appointments.inc.php
M       library/encounter_events.inc.php
A       modules/calendar/drag_copy_event.php
A       modules/calendar/fullcalendar-rightclick-1.9/LICENSE.txt
A       modules/calendar/fullcalendar-rightclick-1.9/README.md
A       modules/calendar/fullcalendar-rightclick-1.9/fullcalendar-rightclick.js
M       modules/calendar/index.php

I'm going to re-revert it somehow..

**Comment by tmccormi** [comment]
This has been reverted (again).   Please make sure your master branches are up-to-date
